### PR TITLE
testBROWSE_LANGUAGES_M_NO added

### PR DESCRIPTION
**Pull request by NataliiaOliver**
testBROWSE_LANGUAGES_M_NO added

### CODE REVIEW

**Review by UmidaKaharova — APPROVED** [review]

File: src/test/java/GroupTeamRocketTest.java [hunk 1]
```diff
@@ -451,4 +453,17 @@ public void testSergeDotMenuStartTitle() {
...
+    @Test
+    public void testBROWSE_LANGUAGES_M_NO() {
+        String expectedResult = "MySQL";
+
+        getDriver().get("http://www.99-bottles-of-beer.net/");
+        getDriver().findElement(By.xpath("//ul[@id='menu']/li/a[@href='/abc.html']")).click();
+        getDriver().findElement(By.xpath("//a[@href='m.html']")).click();
+
+        String actualResult = getDriver().findElement(By.xpath("//tr[last()]/td/a")).getText();
+
```
Comment (Folevald) [inline src/test/java/GroupTeamRocketTest.java:466]: Try to keep the code size (empty strings)

**Review by Folevald — APPROVED** [review]
Try to get size of the code. (empty strings)

**Comment by NataliiaOliver** [comment]
Thank you for your approves!

File: src/test/java/GroupTeamRocketTest.java (hunk 1 above)
Comment (NataliiaOliver) [reply src/test/java/GroupTeamRocketTest.java:466]: Thank you. Will do

**Pull request merged**